.github/workflows/tests.yml

```diff
@@ -11,14 +11,17 @@ on:
       - reopened
       - synchronize
       - ready_for_review
-    paths:
-      - '**/*.py'
-      - '**/*.yml'
-      - '**/*.ipynb'
-      - '**/setup.cfg'
-      - '**/pyproject.toml'
-      - '.github/**/*'
-      - '!tutorials/*'   # This path is excluded, see tutorials.yml
+
+    # Disabled due to branch protection rules. 
+    # PRs affecting paths not listed here, or tutorials/, are otherwise unmergeable.
+    # paths:
+    #   - '**/*.py'
+    #   - '**/*.yml'
+    #   - '**/*.ipynb'
+    #   - '**/setup.cfg'
+    #   - '**/pyproject.toml'
+    #   - '.github/**/*'
+    #   - '!tutorials/*'  # See if it's possible to require tutorials.yml on PRs affecting this path.
```

Comment:
I think this could become a permanent fix. There would be a minimal annoyance if e.g. if you fix a typo on a tutorial or you touch the annotation tool's Dockerfile you'd have to wait for all the tests but I think it's ok, WDYT?

Reply:
For me it sounds like a decent tradeoff. The CI still takes 30-40 minutes to complete, but I don't see how waiting half a hour for a PR can be an issue 🙂